docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+timesketch:
+    build: .
+    ports:
+        - "5000:5000"
+    links:
+        - elasticsearch
+        - postgres
+    environment:
+        - POSTGRES_USER=timesketch
+        - POSTGRES_PASSWORD=password
+        - TIMESKETCH_USER=admin
+        - TIMESKETCH_PASSWORD=password
+        - POSTGRES_ADDRESS=postgres # Use hostname linking
+        - POSTGRES_PORT=5432
+        - ELASTIC_ADDRESS=elastic # Use hostname linking
+        - ELASTIC_PORT=9200
+    restart: always
+elasticsearch:
+    image: elasticsearch:2.4.1
+    command: elasticsearch -Des.node.name="TimesketchNode" -Des.script.groovy.sandbox.enabled=true
```

Comment:
We will not need this anymore when the new release is out. I have moved to script files on the server instead so we can skip the dynamic scripting all together. We can change this later though.